src/main.c

```diff
@@ -141,22 +142,22 @@ void main_TNTP(int argc, char* argv[]) {
                       Bparameters.demandMultiplier);
    }
 #else
-   readOBANetwork(network, argv[1], argv + 2, argc - 2,
+   readOBANetwork(network, argv[3], argv + 4, atoi(argv[2]),
                   Bparameters.demandMultiplier);
 #endif
 
-   /* Default: one batch */
-   setBatches(network, network->numOrigins, argv[2] == NULL);
-
    displayMessage(FULL_NOTIFICATIONS, "Starting Algorithm B...\n");
-   Bparameters.convergenceGap = 1e-14;
+   Bparameters.convergenceGap = atof(argv[1]);

```

Comment:
While we can make gap configurable, I think it'd make more sense to keep a default value and then let someone change it if they really wanted to.